src/System.Runtime.Extensions/tests/VoidMainWithExitCodeApp/project.json

```diff
@@ -8,6 +8,8 @@
   },
   "frameworks": {
     ".NETCoreApp,Version=v1.0": {},
+    ".NETCoreApp,Version=v1.1": {},
+    ".NETStandard,Version=v1.5": {},
```

Comment:
Why are we using the full name instead of the short name? why not netstandard1.5 and netcoreapp1.0/1.1?